kubernetes/cronjob.yaml

```diff
@@ -10,10 +10,9 @@ spec:
         spec:
           containers:
             - name: hello
-              image: dev-lifebank-hapi
-              imagePullPolicy: IfNotPresent
-              command:
+              image: leister/lifebank-hapi:latest
```

Comment:
Use environmental vars for image 

`${DOCKER_REGISTRY}/${IMAGE_NAME_HAPI}:${VERSION}`

Reply:
@xavier506 
I used my local hapi image only for test purposes, my local image has some new files that eoscostarica506 doesn't has, I soon as a finish the new hapi logic, I'm gonna change it and get in touch with you to make a clean docker image upload to eoscostarica506